DESCRIPTION

```diff
@@ -1,48 +1,52 @@
 Package: conos
 Title: Clustering on Netowrk of Samples
-Version: 1.0.2
-Authors@R: c(person("Peter", "Kharchenko", email = "peter_kharchenko@hms.harvard.edu", role = c("aut", "cre")), 
+Version: 1.0.3.9000
```

Comment:
As you merge it to master, please, set the version to `1.0.3` and add corresponding note to CHANGELOG.md.

Reply:
Done. Let me know if you want me to expand more in the CHANGELOG note

Reply:
It's fine, thanks!